ext/lib/crypto/mbedtls/Kconfig

```diff
@@ -58,6 +58,7 @@ endif
 
 config MBEDTLS_SSL_MAX_CONTENT_LEN
 	int "Max payload size for TLS protocol message"
+	default 768 if NET_L2_OPENTHREAD
```

Comment:
One thing that is a bit controversial, is that now we introduced some networking related options to mbedtls config. Not sure how could we solve this differently.

Reply:
> One thing that is a bit controversial, is that now we introduced some networking related options to mbedtls config. Not sure how could we solve this differently.

agree, we need a better way to do this

Reply:
@nashif @jukkar Do you have any particular solution in mind? The options we have to set this value for OpenThread needs (please correct me if I miss something):
* add `default` value, just as it's done now,
* set this value explicitly in every `overlay-ot.conf` file.

Ideally, it would be best for me if I could modify the default from the OpenThread's Kconfig file. Unfortunately, `select`/`imply` works only with boolean options to my knowledge.

Reply:
Probably from these two options, adding the needed values to overlay-ot.conf file is the best. If we at some date get support for global overlays (see #14740), then we could have just one overlay-ot.conf file and managing these settings would be trivial.